src/main/java/in/projecteka/consentmanager/link/LinkConfiguration.java

```diff
@@ -46,14 +46,16 @@ public Link link(WebClient.Builder builder,
                      CentralRegistry centralRegistry,
                      GatewayServiceProperties gatewayServiceProperties,
                      LinkServiceProperties serviceProperties,
-                     CacheAdapter<String, String> linkResults) {
+                     CacheAdapter<String, String> linkResults,
+                     CacheAdapter<String, String> patientLinkReferenceResults) {
         return new Link(
                 new LinkServiceClient(builder, centralRegistry, gatewayServiceProperties),
                 linkRepository,
                 centralRegistry,
                 serviceProperties,
-                linkResults);
-    }
+                linkResults,
+                patientLinkReferenceResults);
```

Comment:
I think there is no need to pass another CacheAdapter to the Link Service. Use the older linkResults. its ok.

Reply:
done